GitCommands/DiffMergeTools/SemanticMerge.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace GitCommands.DiffMergeTools
+{
+    internal class SemanticMerge : DiffMergeTool
+    {
+        private static readonly string[] Folders = GetFolders();
+
+        /// <inheritdoc />
+        public override string DiffCommand => "-s \"$LOCAL\" -d \"$REMOTE\"";
+
+        /// <inheritdoc />
+        public override string ExeFileName => "semanticmergetool.exe";
+
+        /// <inheritdoc />
+        public override string MergeCommand => "-s \"$REMOTE\" -d \"$LOCAL\" -b \"$BASE\" -r \"$MERGED\"";
+
+        /// <inheritdoc />
+        public override string Name => "semanticmerge";
```

Comment:
semanticdiff for the difftool currently

Reply:
In the latest version there is a single executable, naming them differently is not a good idea.

Reply:
Then this should be documented in commit message and reslease notes

Reply:
This will continue to work as is - configured tools will be recognised even though they may not be in our official list:
![image](https://user-images.githubusercontent.com/4403806/69494267-bbebb680-0f0d-11ea-9783-969d25bf259e.png)

Reply:
This should be documented in the commit:
The diff tool names _semanticdiff_ and _tortoisemerge_ is now named _semanticmerge_ and _tmerge_ as the mergetools already was called.